engine/src/main/java/org/terasology/logic/chat/ChatSystem.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2013 MovingBlocks
+ * Copyright 2016 MovingBlocks
```

Comment:
Feel free to bump all the way to 2017 any time you modify a file with out of date years :D